products/storage/api.yaml

```diff
@@ -780,4 +780,70 @@ objects:
         name: timeUpdated
         api_name: updated
         description: The modification time of the object metadata.
-
+  - !ruby/object:Api::Resource
+    name: 'HmacKey'
+    kind: 'storage#hmacKey'
+    base_url: projects/{{project}}/hmacKeys
+    create_url: projects/{{project}}/hmacKeys?serviceAccountEmail={{serviceAccountEmail}}
+    self_link: projects/{{project}}/hmacKeys/{{accessId}}
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation': 'hhttps://cloud.google.com/storage/docs/authentication/managing-hmackeys'
+      api: 'https://cloud.google.com/storage/docs/json_api/v1/projects/hmacKeys'
+    description: |
+      The hmacKeys resource represents an HMAC key within Cloud Storage. The resource
+      consists of a secret and HMAC key metadata. HMAC keys can be used as credentials
+      for service accounts.
+    identity:
```

Comment:
Ah, looks like InSpec isn't exiting early even though we've applied `exclude` to it, and it's breaking the generator. Can you try moving this `identity` part to `terraform.yaml`? I _think_ that works.

Because `flatten_object` is Terraform-exclusive, InSpec is failing to find an `accessId` field (it sees `metadata.accessId`).

If identity doesn't work in `terraform.yaml`, we can set an `import_format` of `['projects/{{project}}/hmacKeys/{{access_id}}']`